### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Tranquility2**
Added a comment in the README regarding the badge

### CODE REVIEW

**Comment by Tranquility2** [comment]
Hi @petargyurov :)
I really like this visualization and planning to use it (the same way you did on my personal page).
I didn't want to just copy the code, so the best idea I had to give credit and help other use this cool project is using a **badge**, which I'm personally going to use and really hope other will in the future 😄

File: README.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ I originally aimed for a no-JS implementation but there was no way of adding ran
 
 Yep, do whatever you want with it.
 
+If you want you can add this badge to help other know about this cool project:  
+[![Using Virtual Bookshelf](https://img.shields.io/badge/Extended%20with-Virtual%20Bookshelf-blue?logo=github)](https://github.com/petargyurov/virtual-bookshelf)
+
```
Comment (petargyurov) [inline README.md]: Nice 😄 I would probably re-word it slightly. How about this?

> If you found this project useful you can make use of the following badge to spread the word

**Review by petargyurov — CHANGES_REQUESTED** [review]
Great idea! I think I would just re-word the sentence slightly.

File: README.md (hunk 1 above)
Comment (Tranquility2) [reply README.md]: Sounds great, will update.

**Review by petargyurov — APPROVED** [review]

**Pull request merged**